static/js/exchange.js

```diff
@@ -49,7 +49,7 @@ updateExchange()
 const tableOptions = {
   dataSource: new UrlDataSource(exchangeUrl + '/bindings/source', { useQueryState: false }),
   pagination: true,
-  keyColumns: ['properties_key']
+  keyColumns: ['properties_key', 'destination', 'routing_key']
```

Comment:
Are you sure `routing_key` is required? `properties_key` should already include the routing key: https://github.com/cloudamqp/lavinmq/blob/9c2ae1ce4089630479e9e67b919ee84d8d0ae593/src/lavinmq/exchange/exchange.cr#L233-L244 

```suggestion
  keyColumns: ['destination', 'properties_key']
```

Reply:
yes you're right! 
why put properties_key last in the array?